client/.gitignore

```diff
@@ -17,6 +17,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+**/components/ComponentSandbox.tsx
```

Comment:
what is it?

Reply:
Just a sandbox to test component render when we code style, instead doing this in an existing page